lua/telescope/builtin/init.lua

```diff
@@ -175,6 +175,7 @@ builtin.git_bcommits = require("telescope.builtin.git").bcommits
 ---   - `<C-r>`: rebases currently selected branch
 ---   - `<C-a>`: creates a new branch, with confirmation prompt before creation
 ---   - `<C-d>`: deletes the currently selected branch, with confirmation prompt before deletion
+---   - `<C-m>`: merges the currently selected branch, with confirmation prompt before deletion
```

Comment:
You should only change the mapping for docs here from `<C-m>` to `<C-y>`. telescope.txt is auto-generated from the annotations with CI.

Not sure, you might have git conflicts now at your hand if docgen is triggered :sweat_smile:

Reply:
he should just change it everywhere. CI isnt running thats why i manually ran it. Sometimes that happens for new contributors.

He should also change it in init.lua so it doesnt break in future PRs.

Reply:
I updated it, lets see if conflicts arise or not :crossed_fingers: